addons/web/models/models.py

```diff
@@ -260,6 +261,10 @@ def group_id_name(value):
 
             records = Comodel.with_context(hierarchical_naming=False).search_read([], field_names)
 
+            forest = defaultdict(lambda: {
```

Comment:
lol, tree is clearer ? but congrats on the pun !

Reply:
flat_forest then (we have many trees, many roots)

Reply:
The term forest is common I think

Reply:
or acyclic directed graph maybe?